src/service.ts

```diff
@@ -22,3 +23,17 @@ export const getDates = async () => {
         alert(err);
     }
 };
+
+export const getStackedChart = async (options: {
+    contact: string;
+    type: string;
+}): Promise<StackedChartData[] | undefined> => {
+    try {
+        const response = await axios.get(`${backendUrl}/stackedchart`, {
```

Comment:
nit: better using `foo-bar` for URL. https://stackoverflow.com/a/18450653/3127828